v2/internal/attacktechniques/azure/persistence/bastion-shareable-link/main.go

```diff
@@ -0,0 +1,182 @@
+package azure
+
+import (
+	"context"
+	_ "embed"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus"
+	"github.com/datadog/stratus-red-team/v2/pkg/stratus/mitreattack"
+	"log"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v6"
+	"fmt"
+)
+
+//go:embed main.tf
+var tf []byte
+
+func init() {
+	const codeBlock = "```"
+	stratus.GetRegistry().RegisterAttackTechnique(&stratus.AttackTechnique{
+		ID:           "azure.persistence.bastion-shareable-link",
```

Comment:
something more action oriented like the below might work better?

```suggestion
		ID:           "azure.persistence.create-bastion-sharing-link",
```

Reply:
Keeping "shareable" if we can as this is the feature name, but can use "sharing" if we're up against a character limit.

Reply:
ah makes sense, so wdyt of `azure.persistence.create-bastion-shareable-link`?